spec/controllers/orga/community_controller_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'spec_helper'
+
+RSpec.describe Orga::CommunityController do
+  render_views
+
+  context 'with admin logged in' do
+    include_context 'with admin logged in'
+    describe 'PUT reset_user_availability' do
+
+      it 'reset availability for users interested in coaching' do
+        put :reset_user_availability
+        expect(response).to redirect_to (orga_users_info_path)
```

Comment:
I'm missing an expectation on the side effects of this request here 👮  My suggestion would be to have something like:
```ruby
expect { put :reset_user_availablity }.to change { user.reload.availablity }.to(false)
```